src/main/java/org/apache/kafka/connect/redis/RedisSourceConfig.java

```diff
@@ -39,15 +41,18 @@
 
 
     public static final ConfigDef CONFIG_DEF = new ConfigDef()
-            .define(HOST, ConfigDef.Type.STRING, "localhost", ConfigDef.Importance.HIGH, "MongoDB Server host")
-            .define(PORT, ConfigDef.Type.INT, 6379, ConfigDef.Importance.HIGH, "MongoDB Server port")
+            .define(HOST, ConfigDef.Type.STRING, "localhost", ConfigDef.Importance.HIGH, "Redis Server host")
+            .define(PORT, ConfigDef.Type.INT, 6379, ConfigDef.Importance.HIGH, "Redis Server port")
+            .define(PASSWORD, ConfigDef.Type.STRING, null, ConfigDef.Importance.LOW, "Redis server password (requirepass setting in redis.conf)" )
+            .define(DB_NAME, ConfigDef.Type.STRING, "0", ConfigDef.Importance.HIGH, "Redis server database name (this is often just 0)")
             .define(TOPIC, ConfigDef.Type.STRING, "redis", ConfigDef.Importance.HIGH, "Redis database topic in kafka")
             .define(POLL_BATCH_SIZE, ConfigDef.Type.INT, 100, ConfigDef.Importance.HIGH, "Task poll batch size")
-            .define(IN_MEMORY_EVENT_SIZE, ConfigDef.Type.LONG, 1024L, ConfigDef.Importance.HIGH, "In memory event size")
+            .define(IN_MEMORY_EVENT_SIZE, ConfigDef.Type.LONG, 209715200L, ConfigDef.Importance.HIGH, "In memory event size")//200 MB default im-memory queue size
```

Comment:
why was the default changed?

Reply:
I noticed that with fairly normal volume (one write every 2 seconds), I was receiving errors. The default seems very small (only 1 KB).